src/Microsoft.Azure.WebJobs.Host/Scale/ScaleOptions.cs

```diff
@@ -78,20 +80,27 @@ public string Format()
             var options = new JObject
             {
                 { nameof(ScaleMetricsMaxAge), ScaleMetricsMaxAge },
-                { nameof(ScaleMetricsSampleInterval), ScaleMetricsSampleInterval }
+                { nameof(ScaleMetricsSampleInterval), ScaleMetricsSampleInterval },
+                { nameof(MetricsPurgeEnabled), MetricsPurgeEnabled },
+                { nameof(IsTargetScalingEnabled), IsTargetScalingEnabled }
             };
 
             return options.ToString(Formatting.Indented);
         }
 
         /// <summary>
-        /// Gets or sets if target base scale is enabled.
+        /// Gets or sets a value indicating whether target base scaling is enabled at the host level.
         /// </summary>
-        public bool IsTargetBasedScalingEnabled { get; set; }
+        public bool IsTargetScalingEnabled { get; set; }
 
         /// <summary>
-        /// Gets or sets the function to checks if target scaler is supported for specific <see cref="ITargetScaler"/>
+        /// Gets or sets a function to check if a particular <see cref="ITargetScaler"/> is enabled.
         /// </summary>
-        public Func<ITargetScaler, bool> IsTargetBasedScalingEnabledForTriggerFunc { get; set; }
+        public Func<ITargetScaler, bool> CheckTargetScalerEnabled { get; set; }
```

Comment:
Because we only have a temporary need for this while we're doing staged rollout, I'm hesitant to add a first class API for it. I wonder instead if we don't just need a general feature flags API that we can use for this and other things?